### PR TITLE
Update component “dropdown-multi-selects”

### PR DESCRIPTION
**Pull request by dannoye**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/_areas/design-system/components/dropdown-multi-selects.md [hunk 1]
```diff
@@ -1,220 +1,132 @@
...
 
-A dropdown multi-select is a type of input field in which a user fills an input box with one or many items from a list of acceptable inputs.
+- - -
 
----
-
-## When to use
+## Usage
 
 Appropriate for lists larger than 2 options that may require a user to select more than 1 option.
```
Comment (itsmemin) [inline src/_areas/design-system/components/dropdown-multi-selects.md:21]:
```suggestion
A form control used when there are more than 2 options and may require a user to select more than 1 option.
```

File: src/_areas/design-system/components/dropdown-multi-selects.md [hunk 2]
```diff
@@ -1,220 +1,132 @@
...
-
-An option that is visibly selected.
+The anatomy of a dropdown multi-select.
 
-#### Options list
+![Dropdown multi-select anatomy](/images/multi_select_anatomy.svg)
 
-The grouping of related input options.
+1. **Input:** The field designated for selecting an individual option from a grouping of related options.
+2. **Chevron:** The chevron arrow, within the input suggesting that there are items to select within a dropdown.
```
Comment (itsmemin) [inline src/_areas/design-system/components/dropdown-multi-selects.md:34]:
```suggestion
2. **Chevron:** The chevron icon indicates there are options available in a dropdown list.
```

File: src/_areas/design-system/components/dropdown-multi-selects.md [hunk 3]
```diff
@@ -1,220 +1,132 @@
...
-<GridCol col="span-6">
+<br>
 
 #### Icons
 
-![Spatial specs for options lists with icons next to options](/images/components/dropdown-multi-select/multiselect_icon_specs.svg)
-
-- Use icons to distinguish between options.
-- Often icons are used when you have two different types of options in a list.
+* Use icons to distinguish between options.
```
Comment (itsmemin) [inline src/_areas/design-system/components/dropdown-multi-selects.md:58]:
```suggestion
Use icons to distinguish between option types.
```

File: src/_areas/design-system/components/dropdown-multi-selects.md [hunk 4]
```diff
@@ -1,220 +1,132 @@
...
+* Often icons are used when you have two different types of options in a list.
```
Comment (itsmemin) [inline src/_areas/design-system/components/dropdown-multi-selects.md:59]: Remove this line

File: src/_areas/design-system/components/dropdown-multi-selects.md [hunk 5]
```diff
@@ -1,220 +1,132 @@
...
 
-- Normal behavior is to reveal the list down when placement of the select input is high enough within the viewport. When select input is lower on the screen the list will reveal up.
-- Dropdrown chevron icon: rotates 180 degrees clockwise when opened, when closed spins back 180 degrees counter clockwise, returning to the original position.
+- - -
 
-</GridCol>
+## Behavior
 
-<GridCol col="span-6">
+Sort the list in a logical order. For instance, put the most selected option(s) at the top. In many cases we recommend alphabetical order.
```
Comment (itsmemin) [inline src/_areas/design-system/components/dropdown-multi-selects.md:81]:
```suggestion
#### Options list
Sort the list in a logical order. For instance, alphabetical order or put the most selected option(s) at the top.
```

File: src/_areas/design-system/components/dropdown-multi-selects.md [hunk 6]
```diff
@@ -1,220 +1,132 @@
...
 
-### Open / close
+![Dropdown multi-select list states](/images/multiselect_list_states.svg)
 
-- Normal behavior is to reveal the list down when placement of the select input is high enough within the viewport. When select input is lower on the screen the list will reveal up.
-- Dropdrown chevron icon: rotates 180 degrees clockwise when opened, when closed spins back 180 degrees counter clockwise, returning to the original position.
+- - -
 
-</GridCol>
+## Behavior
```
Comment (itsmemin) [inline src/_areas/design-system/components/dropdown-multi-selects.md:79]: Some do/do not examples would be great in this section!

File: src/_areas/design-system/components/dropdown-multi-selects.md (hunk 4 above)
Comment (itsmemin) [reply src/_areas/design-system/components/dropdown-multi-selects.md:54]: The second line

File: src/_areas/design-system/components/dropdown-multi-selects.md [hunk 7]
```diff
@@ -1,220 +1,231 @@
...
 
----
+[See typeahead](/design-system/patterns/typeahead/)
 
-## Organizing the list
+#### Defaulting a selection
 
-Sort the list in a logical order. For instance, put the most selected option(s) at the top. In many cases we recommend alphabetical order.
+* A dropdown select shouldn't default a selected option. If you're confident users will likely choose an option you may consider defaulting to a selection.
+* Sometimes choosing 'none' as an option is necessary, if none of the options are applicable to the user. None should appear at the top of the list.
```
Comment (juliexiong) [inline src/_areas/design-system/components/dropdown-multi-selects.md:102]:
```suggestion
* Sometimes choosing "None" as an option is necessary if no options are applicable to the user when a Dropdown multi-select is required. "None" should appear at the top of the list.
```
This is a little odd to point out, but if it's a frequent UX question, may be nice to elaborate. Should "None" only be applicable when the dropdown multi-select is a required field? What if this field isn't required at all? Users should be able to leave it blank without selecting "None".

File: src/_areas/design-system/components/dropdown-multi-selects.md [hunk 8]
```diff
@@ -1,220 +1,231 @@
...
 
-<Grid>
+#### Selection wrapping
 
-<GridCol col="span-6">
+* When multiple options are selected in a multi-select list, selections will populate the input and eventually will have to wrap, in order to fit.
```
Comment (juliexiong) [inline src/_areas/design-system/components/dropdown-multi-selects.md:106]:
```suggestion
* When options are selected from the Dropdown multi-select option list, selections will populate inside the select box. Multiple selections should wrap within the select box to fit all.
```

File: src/_areas/design-system/components/dropdown-multi-selects.md [hunk 9]
```diff
@@ -1,220 +1,231 @@
...
+![Tags selection wrapping](/images/tag_specs.svg)
 
-![Spatial specs for options lists with grouping](/images/components/dropdown-multi-select/multiselect_grouping_specs.svg)
+#### Removing the selection
 
-- Sometimes you'll need to add grouping to a long list of options. Titles can break up options into groups to help users more easily locate them.
+* After a user has chosen an option it turns into a selection within the input. Users can remove a selection by using one of three methods.
+* Selections can be removed by clicking on the X icon within the selection.
+* Backspacing over a selection in the input removes it.
+* By deselecting the checkbox within an option list will also remove the selection.
```
Comment (juliexiong) [inline src/_areas/design-system/components/dropdown-multi-selects.md:115]:
```suggestion
* After a user has chosen an option it appears as a selection within the select box. Users can remove a selection by using one of three methods:
  * Selections can be removed by clicking on the X icon within the selection.
  * Backspacing over a selection in the select box removes it.
  * Deselecting the checkbox within the option list will also remove the selection.
```

**Pull request merged**